angel_system/data/common/load_data.py

```diff
@@ -29,6 +29,14 @@ def sanitize_str(str_: str):
     """
     return str_.lower().strip(" .")
 
+def time_from_name(fname, topic="cooking"):
```

Comment:
add some assertion that the fname matches the RE_FILENAME_TIME parsing that we expect - if it doesn't, print RE_FILENAME_TIME